testing/test_dhcp.sh

```diff
@@ -114,10 +123,8 @@ for iface in $(seq 1 5); do
     else
       echo "Device $iface ip triggers: $((ip_triggers > 0)) $((long_triggers > 0))" | tee -a $TEST_RESULTS
     fi
+
 done
 
-dhcp_timeouts=$(cat inst/cmdrun.log | fgrep 'DHCP times out after 120s lease time' | wc -l)
-device_6_dhcp_timeouts=$(cat inst/cmdrun.log | fgrep 'DHCP times out after 120s lease time' | fgrep 'ipaddr_ipaddr06' | wc -l)
 echo "DHCP timeouts: $dhcp_timeouts" | tee -a $TEST_RESULTS
```

Comment:
This can be removed.

Reply:
@henry54809 were you referring to the total DHCP timeouts?

Reply:
That check was supposed to be for device 6, but you have included device 6 results in the for loop before this already.